client/landing/stepper/declarative-flow/internals/steps-repository/design-choices/index.tsx

```diff
@@ -91,66 +68,17 @@ const DesignChoicesStep: StepType< { submits: { destination: string } } > = ( {
 		submit?.( { destination } );
 	};
 
-	const getCreateWithAILabel = () => {
-		if ( hasEnTranslation( 'Create with AI {{small}}(BETA){{/small}}' ) ) {
-			return translate( 'Create with AI {{small}}(BETA){{/small}}', {
-				components: {
-					small: <span className="design-choices__beta-label" />,
-				},
-			} );
-		}
-
-		if ( hasEnTranslation( 'Create with AI' ) ) {
-			return translate( '%s {{small}}(BETA){{/small}}', {
-				args: [ translate( 'Create with AI' ) ],
-				components: {
-					small: <span className="design-choices__beta-label" />,
-				},
-				comment: 'Do not translate',
-			} );
-		}
-
-		return translate( 'Create with AI (BETA)' );
-	};
-
-	const bigSkyBadgeLabel = () => {
-		if ( ! isLoading && isEligible && personalProduct?.cost_per_month_display ) {
-			if ( hasEnTranslation( 'Starting at %(price)s/month' ) ) {
-				return translate( 'Starting at %(price)s/month', {
-					args: { price: personalProduct.cost_per_month_display },
-					comment: 'Translators: "price" is a per month price and includes a currency symbol',
-				} );
-			}
-
-			return translate( 'Starting at %(price)s a month', {
-				args: { price: personalProduct.cost_per_month_display },
-				comment: 'Translators: "price" is a per month price and includes a currency symbol',
-			} );
-		}
-
-		return undefined;
-	};
-
```

Comment:
In https://github.com/Automattic/wp-calypso/pull/101921#discussion_r2021709432, @paulopmt1   mentioned that [there are plans to re-use this screens](https://github.com/Automattic/wp-calypso/issues/102252)

However, I don't have the capacity to work on that task (I'm mostly working on code cleanups as a side-gig).

Also, if we're not certain about when the task will be implemented, my recommendation is to delete the code. We can always add a reference to this PR, should we want to re-use the code being deleted.